src/clients/c++/request.h

```diff
@@ -329,6 +329,13 @@ class InferContext {
     /// \param input The vector holding tensor string values.
     /// \return Error object indicating success or failure.
     virtual Error SetFromString(const std::vector<std::string>& input) = 0;
+
+    // Set the shard memory object for the batch tensor input
+    /// \param shm_key The shared memory location holding the input tensors.
+    /// \param offset The offset for the batch input tensor.
+    /// \param byte_size The byte size of the batch input tensor.
```

Comment:
"The size, in bytes of the input tensor data. Must match the size expected by the input".